gooddata-sdk/gooddata_sdk/catalog.py

```diff
@@ -17,99 +17,55 @@
 
 
 class CatalogEntry:
+    def __init__(self, entity: dict[str, Any]) -> None:
+        self._e = entity["attributes"]
+        self._entity = entity
+        self._obj_id = ObjId(self._entity["id"], type=self._entity["type"])
+
     @property
     def id(self) -> str:
-        raise NotImplementedError()
+        return self._entity["id"]
 
     @property
     def type(self) -> str:
-        raise NotImplementedError()
-
-    @property
-    def obj_id(self) -> ObjId:
-        raise NotImplementedError()
+        return self._entity["type"]
 
     @property
     def title(self) -> str:
-        raise NotImplementedError()
+        return self._e.get("title")
```

Comment:
If get is used then return value can be also None if key is not presented in dictionary - Optional[str].

Reply:
Optional added